File: applications/advection/2d/swirl_mapped/mapc2m.f
      subroutine mapc2m(blockno,xc,yc,xp,yp,zp,alpha)
     &      bind(c,name="mapc2m")
      implicit none
      integer blockno
      double precision xc,yc,xp,yp,zp,alpha

      call mapc2m_fivepatch(blockno,xc,yc,xp,yp,zp,alpha)

      end

File: applications/advection/2.5d/swirl/mapc2m.m
function [xp,yp,zp] = mapc2m(xc,yc)

map = 'fivepatch';
% map = 'cart';

switch map
    case 'cart'
        xp = xc;
        yp = yc;
        zp = 0*xp;
    case 'fivepatch'
        [xp,yp,zp] = mapc2m_fivepatch(xc,yc);
end


end